kuzzleio/kuzzle
# Add document global events (database:*)

## What does this PR do ?

- add document global event

### How should this be manually tested?

- create a plugin pipes `database:beforeWrite`
- call action `document:create`
- see your pipe called with documents

### Other changes

/

### Boyscout

- fix a possible error in a functional test
```javascript
else if(response.errors === null && withErrors) { // response.error not errors
  callback('Should get partial error');
   return false;
}
```
